tests/installation/yast2_firstboot.pm

```diff
@@ -92,9 +77,16 @@ sub firstboot_registration {
     $firstboot->press_next();
 }
 
+sub firstboot_language_keyboard {
+    wait_still_screen();
+    my $default_language = get_test_suite_data()->{default_language};
+    my $firstboot        = $testapi::distri->get_firstboot();
+    $firstboot->setup_language_and_keyboard($default_language);
+}
+
 sub firstboot_keyboard {
+    wait_still_screen();
```

Comment:
Out of curiosity, did you try to use `is_shown`, does that help to make it work properly?

Reply:
I am using it for the debug_label lb_main, but I didn't try it for the value. If I use it to check if the value exists, it will not matter if it's Afrikaan or English, so it wouldn't save from failure. You have something else in mind?

Reply:
As @JRivrain  has raised, `debug_label` doesn't work as expected and always returns true, even if element is not there. So we could use comboboxes to make sure that correct page is shown.